frontend/src/utils/request.ts

```diff
@@ -1,15 +1,15 @@
 // RequestState machinery
 // This file might be removed if we start using redux-query or recoil
 
-export type RequestState<T>
-  = { status: "idle" }
+export type RequestState<T> =
+  | { status: "idle" }
   | { status: "loading" }
-  | { status: "succeeded", result: T }
-  | { status: "failed", error?: string };
+  | { status: "succeeded"; result: T }
```

Comment:
how I sick and tired of this bullshit. why is it even important? 

I get used to commas in types, you don't even write frontend but set up some rules how it should look like. why it matters? 

total crap, wasting of time.

nvm, it's just shit which I should swallow and then I will revert it back when your attention will be focused on tight deadlines and not on this unessential miserable rules.

Reply:
That's actually just `prettier`. You can just set your editor up to prettify your code on save and forget about it forever.

Reply:
I did it, but the main issue that my ide is highlighting  all eslint problems on fly and the code is just highlighted red, I am scared/confused, and I have to tolerate this while I end up with some compiled code, then it might be finally prettified. So it just makes code writing harder to me.